pac/tasks/update-deployment-patch.yaml

```diff
@@ -30,6 +30,13 @@ spec:
           value: $(params.image)
       script: |-
         #!/bin/bash
+        # check if the updated deployment is a bearer authentication case.
+        INCLUDE_MODEL_ENDPOINT_SECRET=$(kubectl get configmap ${DEPLOYMENT_NAME}-app-config -n ${DEPLOYMENT_NAMESPACE} -o jsonpath="{.data.INCLUDE_MODEL_ENDPOINT_SECRET}")
```

Comment:
what is adding these the key/value pairs to the config map? i.e. `.data.INCLUDE_MODEL_ENDPOINT_SECRET`

Reply:
Those three values are added in the config map here: https://github.com/redhat-ai-dev/ai-lab-helm-charts/blob/main/charts/ai-software-templates/chatbot/templates/app-config.yaml#L21-L23

Reply:
cool ... add a comment here explaining that

Reply:
Added an extra comment in [88d5e25](https://github.com/redhat-ai-dev/rhdh-pipelines/pull/11/commits/88d5e25144b046c00db3c26a0bcfa7ee6fecf0ff)